sources/install/package_web.sh

```diff
@@ -894,6 +894,19 @@ function install_postman() {
     add-history postman
     add-test-command "which postman"
     add-to-list "postman,https://www.postman.com/,API platform for testing APIs"
+
+function install_zap() {
+    colorecho "Installing ZAP"
+    mkdir /opt/tools/ZAP
+    # using $(which curl) to avoid having additional logs put in curl output being executed because of catch_and_retry
+    zap_version=$($(which curl) -s "https://www.zaproxy.org/download/" | grep -G -o "ZAP \d+(\.\d+)+" | head -1 | cut -d' ' -f2)
```

Comment:
Can you replace `$(which curl)` with `curl` ?

Reply:
I have no output with this grep command :

```bash
exegol-test /workspace # curl -s "https://www.zaproxy.org/download/"| grep -G -o "ZAP \d+(\.\d+)+"

exegol-test /workspace #
```

Reply:
Oh yeah sure sorry, it should have been extended regex, grep has some difficulty like that.
For the curl, I changed has requested but I simply copy the way burp works
`    # using $(which curl) to avoid having additional logs put in curl output being executed because of catch_and_retry`